Formula/mpage.rb

```diff
@@ -0,0 +1,21 @@
+class Mpage < Formula
+  desc "Mpage, many to one page printing utility"
+  homepage "http://www.mesa.nl/pub/mpage/"
+  url "http://www.mesa.nl/pub/mpage/mpage-$version.tgz"
+  version "2.5.6"
```

Comment:
> I'm pretty sure I installed the formula with this syntax.

You must have had a good URL to begin with, downloaded with that URL, then switched to the bad one. Later when you run `brew install` it just picks up the downloaded file from cache and no longer checked that URL. `brew audit --new-formula` can catch that, though.